src/components/slide/index.css

```diff
@@ -20,20 +20,27 @@ p {
 }
 
 .invitation-wrapper p {
+    font-size: 1.5em;
     font-weight: bold;
 }
 
 .question-wrapper {
     display: flex;
-    align-items: center;
+    align-items: flex-start;
     justify-content: flex-start;
-    margin-top: 1.2em;
+    margin-top: 4em;
```

Comment:
Do you re-check the ui of slide item and slide preview in presentation detail page ? @dlxkhang